packages/react-components/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -381,19 +372,19 @@ const useRootFocusStyles = makeStyles({
 const useRootIconOnlyStyles = makeStyles({
   // Size variations
   small: {
-    ...shorthands.padding('4px'),
+    ...shorthands.padding(tokens.spacingHorizontalXS),
```

Comment:
These ones too should use vertical and horizontal tokens:
```suggestion
    ...shorthands.padding(tokens.spacingVerticalXS, tokens.spacingHorizontalXS),
```